packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -310,7 +310,7 @@ class NetworkRequestsTable extends StatelessWidget {
 
 class UriColumn extends ColumnData<NetworkRequest>
     implements ColumnRenderer<NetworkRequest> {
-  UriColumn() : super.wide('Uri');
+  UriColumn() : super.wide('Uri', minWidthPx: 50);
```

Comment:
Fyi @kenzieschmoll gave this one a minWidth because completely hiding the Uri column when there is not enough room like we were doing is probably exactly the opposite of what we want. 
A larger min width could also be reasonable. Open to suggestions.

Reply:
Could we use a ... in the middle of URI with a hover over to show to the complete URI?

Reply:
if we had a good generic text widget like that then yes that would probably be useful.